### DISCUSSION TITLE
Font: release vs. preview?

### DISCUSSION THREAD

**Discussion by vefatica** [microsoft/terminal#15396] [Q&A]
I have unpackaged installations of the latest release and preview versions, known (in most places) via AppPaths subkeys WT.EXE and WTP.EXE.  They use the same settings file, namely, `C:\Users\vefatica\AppData\Local\Microsoft\Windows Terminal\settings.json`.  My default profile is configured to use Consolas, size 13.  When I start both (from Explorer's Run dialog), I see this.  The sizes of what's rendered are different!  What's going on?  Thanks.

![image](https://github.com/microsoft/terminal/assets/61856645/c8d640c3-3c60-41fc-b0f1-33947a7eb809)

**Comment by vefatica** [comment]
OK, so this is the new rendering engine, off by default in the release version, on by default in the preview.  Once I toggled it in the settings, both versions honor it.  I reckon that's when the setting got into the settings file.

But I must ask ... is the size difference expected/by design?

**Reply by zadjii-msft** [reply]
Pretty sure it is. There's been discussions about the difference in the font size previously on the repo. 

The new renderer is being replaced by an _even more newer_ renderer in 1.18 - I can't remember if the font size in that one is the same as the "DxEngine" or the v1 "AtlasEngine"

**Comment by vefatica** [comment]
WT is included in Windows 11 ... right? ... auto-updated?  Won't there be a few who use it, don't  pay attention to its development, and are distressed by changes in its size (and shape)?  Will the user have the choice of three (and counting) rendering engines?  I don't really care but I'm curious about how you see these things.

I know it has been (and should be) discussed elsewhere, but command-line-specified settings files would facilitate A/B comparisons of a features like this.  I guess I'll write a script to start WT after swapping settings files as specified.

**Reply by zadjii-msft** [reply]
There sure have been. 
* https://github.com/microsoft/terminal/issues/13936#issuecomment-1250180290
* https://github.com/microsoft/terminal/issues/14165

The setting is always there for those that want to go back to the old renderer, and we're always adding more settings. The line spacing settings in 1.18 should let people tweak the fonts to absolutely anything they want.

**Comment by vefatica** [comment]
Thanks for the links.  I haven't yet mastered searching the issues.  😁

I agree.  Being able to tweak line spacing will be nice.  Is that in the 1.17 preview?
